src/workerd/api/gpu/gpu-utils.c++

```diff
@@ -41,7 +41,38 @@ wgpu::FeatureName parseFeatureName(GPUFeatureName &str) {
     return wgpu::FeatureName::Float32Filterable;
   }
 
-  KJ_FAIL_REQUIRE("unknown GPU feature", str);
+  JSG_FAIL_REQUIRE(TypeError, "unknown GPU feature", str);
+}
+
+kj::Maybe<GPUFeatureName> getFeatureName(wgpu::FeatureName &feature) {
```

Comment:
Super minor style nit: Generally we prefer left-leaning `&`s. So...
```suggestion
kj::Maybe<GPUFeatureName> getFeatureName(wgpu::FeatureName& feature) {
```

Reply:
Hum, this is problematic because `clang-format` seems to prefer right-leaning and this pattern is all over the code I wrote... Can we update https://github.com/cloudflare/workerd/blob/main/.clang-format with the format we want?

Maybe we can table this for when we have a solution for auto-formatting the code?

Reply:
I'm going to add `PointerAlignment: Left` to the config.

Reply:
Done.